src/chain/bls_emu.rs

```diff
@@ -8,10 +8,7 @@
 
 //! Types emulating the BLS functionality until proper BLS lands
 use super::{ProofSet, SectionInfo};
-use crate::{
-    id::{FullId, PublicId},
-    QUORUM_DENOMINATOR, QUORUM_NUMERATOR,
```

Comment:
Hmm...   We have the claim that `Quorum is defined as having strictly greater than QUORUM_NUMERATOR / QUORUM_DENOMINATOR`
If at this place, we only use it as 1/3, maybe we shall set the PR title to be:
change the threshold in BLS to just one third ...   ?